hm_pyhelper/gateway_grpc/client.py

```diff
@@ -0,0 +1,231 @@
+import base58
+import grpc
+import subprocess
+import json
+
+from hm_pyhelper.protos import blockchain_txn_add_gateway_v1_pb2, \
+    local_pb2_grpc, local_pb2, region_pb2
+from hm_pyhelper.gateway_grpc.exceptions import GatewayMalformedAddGatewayTxn
+
+from hm_pyhelper.logger import get_logger
+
+LOGGER = get_logger(__name__)
+
+
+def decode_pub_key(encoded_key: bytes) -> str:
```

Comment:
this functions is as is copy from jsonrpc

Reply:
I was hoping that we will be able to merge this into master and not change all requirements.txt to point to a git branch

Reply:
removed json_rpc files.